batch/parsers/xUnitParser.js

```diff
@@ -1,78 +1,56 @@
-import fs from 'fs';
-import xml2js from 'xml2js';
-import {from, time} from './parsed';
-
-const GITHUB_REPO_URL='https://github.com/';
+import xml2jsPromise from 'xml-to-json-promise';
+import {gatherTestSuiteData, normalizeTime} from './testSuiteParserUtils';
+const GITHUB_HOST = 'https://github.com/';
 
 class XUnitParser {
 	constructor(configuration) {
 		this.configuration = configuration;
 	}
 
-	get now() {
-		return new Date();
-	}
-
-	readFile(xUnitFilePath, callback) {
-		fs.readFile(xUnitFilePath, function (err, data) {
-			callback(err, data.toString());
-		});
+	get xml2JSParser() {
+		return xml2jsPromise;
```

Comment:
This is not an instance variable, don't expose it as a such  :wink:  use `xml2jsPromise` directly, no need to scope it inside the class.